charts/woodpecker/charts/agent/README.md

```diff
@@ -26,6 +26,20 @@ A Helm chart for the Woodpecker agent
 
 ## Values
 
+### NetworkPolicy
+
+| Key | Type | Default | Description |
```

Comment:
Please format using `prettier` (I have just added `prettier` checks for the project in #253)